package.json

```diff
@@ -52,9 +52,7 @@
 		"docker:clean": "yarn docker:compose down --rmi all -v && rm -rf docker/wordpress/* docker/wordpress/.htaccess docker/wordpress-develop/* docker/logs/* docker/data/mysql/*",
 		"docker:env": "node -e \"require('fs').createWriteStream( 'docker/.env', { flags: 'a' } );\"",
 		"docker:update-core-unit-tests": "yarn docker:compose exec wordpress svn up /tmp/wordpress-develop/tests/phpunit/data/ /tmp/wordpress-develop/tests/phpunit/includes",
-		"lint": "yarn lint-es6 && yarn lint-es5",
-		"lint-es5": "eslint _inc/*.js modules -c modules/.eslintrc.js",
-		"lint-es6": "eslint --ext .js --ext .jsx ./*.js _inc/client extensions -c .eslintrc.js",
+		"lint": "eslint --ext .js,.jsx ./*.js _inc/*.js _inc/client extensions modules",
```

Comment:
Curious why `./*.js _inc/*.js` here if we already have `--ext`?

Should this be something like `. _inc extensions modules`?

Reply:
Suppose root level folder covers sub-folders? i.e. no need `_inc/*/**`

Reply:
I think `--ext` means the list of file types to lint, while `./*.js _inc/*.js` are wildcard file paths.

Reply:
Docs suggest that both folders and globs work: https://eslint.org/docs/user-guide/command-line-interface (tho seems to be recommended to have them in quotes to ensure they're the same across environments).

At any rate, no action needed in this PR. :-)

Reply:
@simison: `. _inc extensions modules` seems much better for its simplicity. I'll go ahead and make the change.